test/integration/api/order.spec.js

```diff
@@ -656,6 +656,246 @@ test('updates an order move', async (t) => {
   t.is(orderMove.metadata.test, true)
 })
 
+test('simulates a payment process', async (t) => {
```

Comment:
The following test focuses on a precise cart use case (more than the Order API itself) so that you may want to add this keyword to test name, as in `simulates a payment process based on a shopping cart`.

Reply:
Right, I update the test name.